### DISCUSSION TITLE
Starting Grad School how to keep education benefits this summer?

### DISCUSSION THREAD

**Discussion by sandeepmukh** [community/community#56157] [GitHub Education]
### Hi everyone,

Hi! I just graduated from undergrad, but I am attending the same school for grad school starting this fall. Is there a way I can maintain my education benefits during the summer? Submitting a decision letter didn’t seem to work.

**Comment by samus-aran** [comment]
Hi @sandeepmukh  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by jaimeyfrank** [comment]
I am in the same boat: just finished undergrad and starting grad school. However, I'm not even able to submit an application to get my benefits back.
